src/docs/spec.md

```diff
@@ -386,10 +390,10 @@ In external mode, the mapping set metadata MUST be supplied in a separate YAML f
 Example ([download](https://raw.githubusercontent.com/mapping-commons/sssom/master/examples/external/mp-hp-exact-0.0.1.sssom.tsv)):
 
 ```
-subject_id	predicate_id	object_id	mapping_justification	subject_label	object_label
-HP:0009124	skos:exactMatch	MP:0000003	Lexical	Abnormal adipose tissue morphology	abnormal adipose tissue morphology
-HP:0008551	skos:exactMatch	MP:0000018	Lexical	Microtia	small ears
-HP:0000411	skos:exactMatch	MP:0000021	Lexical	Protruding ear	prominent ears
+subject_id  subject_label   predicate_id	object_id	object label    mapping_justification
+HP:0009124  Abnormal adipose tissue morphology	skos:exactMatch	MP:0000003  abnormal adipose tissue morphology	semapv:LexicalMatching
+HP:0008551  Microtia	skos:exactMatch	MP:0000018  small ears	semapv:LexicalMatching
+HP:0000411  Protruding ears skos:exactMatch	MP:0000021  prominent ears	semapv:LexicalMatching
 ```
```

Comment:
The spacing between fields is inconsistent. Some fields are separated by spaces and some are separated by tabs.